renovate.json

```
@@ -1,40 +1,21 @@
 {
-  "extends": [
-    "config:base",
-    ":rebaseStalePrs",
-    ":automergeMinor",
-    ":label(ready)",
-    "group:linters",
-    ":masterIssue",
-    ":assignee(rarkins)"
-  ],
-  "lockFileMaintenance": {
-    "enabled": true,
-    "schedule": ["at any time"],
-    "stabilityDays": 0,
-    "masterIssueApproval": true
-  },
-  "postUpdateOptions": ["yarnDedupeHighest"],
+  "extends": ["github>renovatebot/presets-internal:node", ":label(ready)"],
```

```suggestion
  "extends": ["github>renovatebot/presets-internal:node"],
```

We don't use this label anymore